.docksal/docksal.env

```diff
@@ -27,4 +27,4 @@ MYSQL_PORT_MAPPING='0:3306'
 
 # Enable/disable xdebug
 # To override locally, copy the two lines below into .docksal/docksal-local.env and adjust as necessary
-XDEBUG_ENABLED=0
+XDEBUG_ENABLED="1"
```

Comment:
I'd leave xdebug disabled by default, since it adds an overhead of about 20% when enabled.